### ISSUE TITLE
PostgreSQL 17.2 installation application stackbuilder error

### ISSUE THREAD

**Issue by zfeld75** [EnterpriseDB/edb-installers#214]
I am trying to run the wizard and when clicking next after selecting my local Postgres install, I get the first error message.
If I then select yes, I get the second error message, and the installation fails.
I am behind a VPN, it is a work computer, and I cannot access the internet without using their VPN.

I actually only need the Application StackBuilder to be able to install  PostGIS. 
So, I tried to install PostGIS independently by downloading it from the PostGIS website using their downloaded postgis-bundle-pg13x64-setup-3.5.0-1.exe.

Problem is that when I attempted to enable the PostGIS on a database using CREATE EXTENSION postgis; 
I get this error
ERROR:  PostGIS built for PostgreSQL 13.0 cannot be loaded in PostgreSQL 17.2

![StackBuilder ss1](https://github.com/user-attachments/assets/a60e06f2-d14f-4509-beb1-cc47074c3a1c)

![StackBuilder ss2](https://github.com/user-attachments/assets/6908c62f-e296-47ae-a2e8-4b7d55bfbb25)

**Comment by mshal199** [comment]
.

**Comment by sandeep-edb** [comment]
EDB doesn't package PostGIS windows installer. The error "ERROR: PostGIS built for PostgreSQL 13.0 cannot be loaded in PostgreSQL 17.2"  is very clear that you installed the PostGIS installer meant to be used for PG 13 and instead you must install the one built for PG17. I found the installer here: https://ftp.postgresql.org/pub/postgis/pg17/v3.5.0/win64/ 

The StackBuilder error seems transient. You may retry

**Comment by zfeld75** [comment]
I understand the The PostGIS  error is not an EDB error. I just mentioned this issue.
In regard to the StackBuilder error it is still there, I tried again.

**Comment by sandeep-edb** [comment]
OK. I don't see that StackBuilder issue at my end. @dpage any idea?

**Comment by dpage** [comment]
No, I can't reproduce it here on Windows 11. Everything worked exactly as expected.

**Comment by zfeld75** [comment]
I don't seem to be the only one having this issue.
It seems to be an ongoing issue for years
See https://stackoverflow.com/questions/63462770/postgresql-stack-builder-installation-certificate-verification-problem-on-window
Where the most recent post was this July (https://stackoverflow.com/a/78787871/398482). 
I actually tried what the poster suggested but didn't work for me

**Comment by MetehanKacar** [comment]
Sorun internet güvenlik seçeneğinin yüksek olmasından veya windowsun tarih ve saatinin yanlış olmasından kaynaklıdır sorunun çözümü çok basittir 
https://www.postgresql.org/ftp/postgis/pg14/v3.4.2/win64/
https://www.postgresql.org/ftp/odbc/releases/REL-16_00_0004/
https://www.postgresql.org/ftp/pgadmin/pgadmin4/v6.21/windows/
https://www.enterprisedb.com/downloads/postgres-postgresql-downloads (ilk önce bunu kurun ve 14.x sürümünü kurun)
gerekli bileşenleri manuel yüklemeniz yeterli olacaktır aynısını diğer sürümler içinde yapabilirsiniz konuyu türkçe hortlatmam biraz komik oldu ama çözemeyen arkadaşlarım için değer
